graphql/schema/schemagen.go

```diff
@@ -700,6 +723,63 @@ func genDgSchema(gqlSch *ast.Schema, definitions []string,
 		}
 	}
 
+	for fname, langTagDgPred := range langTagDgPreds {
+		dgPredAndTags := strings.Split(fname, "@")
+		unTaggedDgPredName := dgPredAndTags[0]
+		tags := dgPredAndTags[1]
```

Comment:
Chance of panic in case @ comes at the end of Dgraph predicate name. See the above comment.